queries/orderbook/barn_batch_rewards.sql

```diff
@@ -313,7 +122,7 @@ price_data AS (
         ap_protocol.price / pow(10, 18) as protocol_fee_token_native_price,
         ap_sell.price / pow(10, 18) as network_fee_token_native_price
     FROM
-        order_surplus AS os
+        order_protocol_fee AS os
```

Comment:
The table should be abbreviated to opf instead of os, and the line above changed accordingly.